tagging/models.py

```diff
@@ -177,10 +178,11 @@ def usage_for_queryset(self, queryset, counts=False, min_count=None):
         Passing a value for ``min_count`` implies ``counts=True``.
         """
         compiler = queryset.query.get_compiler(using=queryset.db)
-        where, params = compiler.compile(queryset.query.where)
+        params = None
```

Comment:
I think this line can be removed - does params really need to be set to `None` here?

Reply:
`params` needs to be at least declared here, otherwise it will not be available in the return statement of this function. Results in `local variable 'params' referenced before assignment`